packages/nvidia_nat_mcp/tests/server/test_tool_converter.py

```
@@ -775,3 +777,133 @@ async def test_non_string_non_dict_result_converted_to_string(self):
+
+        # Act - call with sanitized names (as MCP would pass them)
+        result = await wrapper(**{"from_": "2024-01-01", "cik_A": "ABC"})
+
+        # Assert
+        assert result == "ok"
+        # Verify session_manager.run() was called with properly mapped payload
+        mock_session_manager.run.assert_called_once()
+        call_args = mock_session_manager.run.call_args
+        payload = call_args[0][0]
+        # The Pydantic model should have the original field names
+        assert getattr(payload, "from") == "2024-01-01"  # noqa: W601
```

Ruff (0.15.17)

[warning] Invalid rule code in `# noqa`: W601

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)